### DISCUSSION TITLE
How to disable loopback interface

### DISCUSSION THREAD

**Discussion by advancenXX** [eProsima/Fast-DDS#3993] [Q&A]
Is there any way to disable loopback interface? We find too many PDP packages from 127.0.0.1.

**Comment by JesusPoderoso** [comment]
Hi @advancenXX, there is not a direct configuration to disable loopback interface. You can [configure the transport](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/whitelist.html) you are using adding a different interface in the whitelist, so you do not use the loopback interface.

**Comment by juanlofer-eprosima** [comment]
Hi @advancenXX , for your information the new [blocklist](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/interfaces.html#blocklist) configuration option with the requested feature has recently been introduced.

Cheers!
